app/assets/stylesheets/global.scss

```diff
@@ -86,8 +86,8 @@ button, .button {
 /* For UI elements involving reordering */
 .disabled-arrow {
   filter: #808080; /* IE */
-  -webkit-filter: grayscale(1); /* Chrome, Safari, Opera */
   filter: grayscale(1); /* Edge, Firefox */
+  -webkit-filter: grayscale(1); /* Chrome, Safari, Opera */
```

Comment:
Does it specifically want `-webkit-filter` below the rest? If not, I'd place it above (both things), so it gets overridden by the standards version.

Reply:
Fixed